src/pages/form-validation/table/index.tsx

```diff
@@ -0,0 +1,198 @@
+                                    value={row.max}
+                                    classes={{
+                                        root: classes.textFieldRoot,
+                                    }}
+                                    InputProps={{
+                                        classes: {
+                                            input: classes.skinnyInput,
+                                        },
+                                    }}
+                                />
+                            </TableCell>
+                        </TableRow>
+                    ))}
+                </TableBody>
+            </Table>
+        </TableContainer>
+    );
+
+    const getList = (): JSX.Element => (
+        <>
+            {rows.map((row, index) => (
+                <div key={index} style={{ marginBottom: theme.spacing(2) }}>
+                    <InfoListItem
+                        icon={
+                            <Typography variant={'body1'} style={{ color: theme.palette.text.secondary }}>
+                                #{row.id}
+                            </Typography>
+                        }
+                        title={<Typography variant={'h6'}>{row.name}</Typography>}
+                    />
+                    <InfoListItem
+                        title={'Min'}
+                        rightComponent={
+                            <TextField
+                                variant="filled"
+                                value={row.min}
+                                classes={{
+                                    root: classes.textFieldRoot,
+                                }}
+                                InputProps={{
+                                    classes: {
+                                        input: classes.skinnyInput,
+                                    },
+                                }}
+                            />
+                        }
+                    />
+                    <InfoListItem
+                        title={'Max'}
+                        rightComponent={
+                            <TextField
+                                variant="filled"
+                                value={row.max}
+                                classes={{
+                                    root: classes.textFieldRoot,
+                                }}
+                                InputProps={{
+                                    classes: {
+                                        input: classes.skinnyInput,
+                                    },
+                                }}
+                            />
+                        }
+                    />
+                    <InfoListItem
+                        title={''}
+                        style={{ height: theme.spacing(2), marginBottom: -theme.spacing(2) }}
+                        divider={index === rows.length - 1 ? 'full' : 'partial'}
+                    />
```

Comment:
Feels hacky.... you introduced a negative margin + a blank ILI to do the divider. The end result looks visually good though. Wonder how @joebochill feels about this.

Reply:
Ok so the negative margin wasn't required; but the 16px InfoListItem is still there just so we don't have to do any calculations to figure out how big a partial divider should be.